src/parser/expr.js

```diff
@@ -204,28 +204,28 @@ module.exports = {
       case this.tok.T_NEW:
         return this.next().read_new_expr();
 
-      case this.tok.T_ISSET:
+      case this.tok.T_ISSET: {
```

Comment:
why surrounding with `{` ... `}` ? It it a linter directive or is it better ?

Reply:
Yeah, eslint complains when you put `let`/`const` inside `case`. see https://eslint.org/docs/rules/no-case-declarations